src/DeprecatedClassFacade.php

```diff
@@ -39,17 +39,28 @@ public function __construct() {
 		$this->instance = new static::$facade_over_classname();
 	}
 
+	/**
+	 * Log a deprecation to the error log.
+	 */
+	private static function log_deprecation() {
+		error_log( // phpcs:ignore
+			static::class
+			. ' is deprecated since version '
+			. static::$deprecated_in_version
+			. '. Make sure you are using the undeprecated equivalent which '
+			. 'this deprecation facade wraps.'
+		);
```

Comment:
Can we log the same message that would be displayed by core WP with a replacement available?

https://github.com/WordPress/WordPress/blob/master/wp-includes/functions.php#L4851-L4860

Reply:
Done in a0b1eb24f.

**Before:**

> Automattic\WooCommerce\Admin\Notes\WC_Admin_Note is deprecated since version 1.7.0. Make sure you are using the undeprecated equivalent which this deprecation facade wraps.

**After:**

> Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes::get_notes_count is deprecated since version 1.7.0! Use Automattic\WooCommerce\Admin\Notes\Notes::get_notes_count instead.

Reply:
Like this change Jeff, thank you.